scripts/src/test/org/oppia/android/scripts/RegexPatternValidationCheckTest.kt

```diff
@@ -0,0 +1,81 @@
+package org.oppia.android.scripts
+
+import java.io.File
+import kotlin.test.assertEquals
+import org.oppia.android.scripts.RegexPatternValidationCheck
+import org.junit.Test
+import org.junit.Rule
+import org.junit.Before
+import org.junit.rules.TemporaryFolder
+import org.junit.rules.ExpectedException
+import org.oppia.android.scripts.ScriptResultConstants
+
+class RegexPatternValidationCheckTest {
+
+  @Rule
+  @JvmField
+  public var tempFolder: TemporaryFolder = TemporaryFolder()
+
+  @Rule
+  @JvmField
+  var thrown: ExpectedException = ExpectedException.none()
+
+  @Before
+  fun initTestFilesDirectory() {
+    val testFilesDirectory = tempFolder.newFolder("testfiles")
```

Comment:
This val isn't used. Should we maybe verify that the directory exists here as a sanity check?

Reply:
Apologies! we don't need to save it in a val. Now updated.
We are creating a `testfiles` temporary directory before every test. And in this directory we are keeping all the dynamically generated assets.